vars/pipelineCapsuleSystemTests.groovy

```diff
@@ -3,12 +3,42 @@ void call() {
   pipeline {
     agent none
     stages {
-      stage('Call tests') {
+      stage('List test directories') {
+        agent any
         steps {
-          build(
-            job: 'common/system-tests-wrapper',
-            parameters: collectParams(),
-          )
+          gitClone([
+            url: 'git@github.com:vegaprotocol/system-tests.git',
+            branch: params.SYSTEM_TESTS_BRANCH,
+            credentialsId: 'vega-ci-bot',
+            timeout: 2,
+          ])
+          testDirs = sh (
+            script: "find tests -maxdepth 1 -type d | grep 'tests/'",
+            returnStdout: true
+          ).trim().split('\n').findAll{ it }
+          // split into two arrays of half size of the list
+          suits = testDirs.collate(dirs.size() / 2 + 1)
+        }
+      }
+      stage('Call tests') {
+        parallel {
+          stage('Suit 1') {
+            steps {
+              build(
+                job: 'common/system-tests-wrapper',
+                // exclude
+                parameters: [string(name: 'SYSTEM_TESTS_TEST_DIRECTORY', value: suits[0].join(" "))] + collectParams(),
+              )
+            }
+          }
+          stage('Suit 2'){
+            steps{
+              build(
+                job: 'common/system-tests-wrapper',
+                parameters: [string(name: 'SYSTEM_TESTS_TEST_DIRECTORY', value: suits[1].join(" "))] + collectParams(),
+              )
+            }
+          }
```

Comment:
Currently, we are using `marks` (annotations) to divide tests:
- `smoke`: [dirs](https://jenkins.ops.vega.xyz/job/common/job/system-tests-wrapper/26/testReport/)
- `network_infra_smoke`: [dirs](https://jenkins.ops.vega.xyz/job/common/job/system-tests-wrapper/25/testReport/)
- `full`: [dirs](https://jenkins.ops.vega.xyz/job/common/job/Nightly%20Full%20System%20Test%20Run/327/testReport/)
- `network_infra`: [dirs](https://jenkins.ops.vega.xyz/job/common/job/Nightly%20Full%20System%20Test%20Run%20-%20Network%20Infra/46/testReport/)

You can click `dirs` to see which directories contain tests with a given `mark`.

A combination of existing `marks` and automatically selected directories won't work because sometimes it will produce zero tests.
Also, automatically dividing tests by directories does not split tests evenly by time (some dirs take `20sec` to execute, while others `1h+`).

I believe that it would be better if the division into sub-groups should be done manually.

I have several ideas about how it should be done, but it would be faster to meet and discuss.

Extra info about `marks` (annotations):
- every PR commit to `system-tests` and `vega` repos triggers TWO `capsule system-tests` pipelines: one for each mark: `smoke`, `network_infra_smoke` (both smoke),
- every night, we trigger TWO `capsule system-tests` pipelines: one for each mark: `full`, `network_infra` (both non-smoke)